balancer-labs/frontend-v2
# Add Github Action to build + push Dockerfile

# Description

- Add a new Github Action to build Dockerfile.dev and push it to Dockerhub
- Tag image with 'latest' and current package.json version

This needs a dockerhub login added to settings to work, see checklist. 

## Type of change


## How should this be tested?

- I tested this on my personal fork of this repository. Can see test here: https://github.com/timjrobinson/frontend-v2/pull/1. This successfully tagged and pushed the image to Dockerhub
- I tested running `docker run timbalancer/frontend-v2` locally and it ran correctly.

This can be tested by creating a PR in your own fork of this repository, changing the `on` section, and seeing it build + push there. Or you can test with `docker run timbalancer/frontend-v2`

## Checklist:

- [ ] Create Dockerhub build user and add username/password to secrets.
